runtime/lua/vim/lsp/semantic_tokens.lua

```diff
@@ -419,10 +419,10 @@ function STHighlighter:on_win(topline, botline)
         })
       end
 
-      local ft = vim.bo[self.bufnr].filetype
```

Comment:
was there anything wrong with this line? this is the canonical lua-level API for buffer options.

Reply:
Yes agreed. We should use `vim.bo` here.

Reply:
Done.